provisioner/ansible/provisioner.go

```diff
@@ -382,6 +391,21 @@ func validateFileConfig(name string, config string, req bool) error {
 	return nil
 }
 
+func validateDirectoryConfig(name string, config string, req bool) error {
```

Comment:
What should I put into the error message?  Should I just hardcode `inventory_directory`?